daq/forchestrator.py

```diff
@@ -40,15 +42,27 @@ def _handle_faucet_events(self):
             LOGGER.debug('Faucet event %s', event)
             if not event:
                 return True
+
+            timestamp = event.get("timestamp", time.time())
```

Comment:
Are there actual events where there is no timestmp?  This is fine for now (functional), but we should likely keep in mind that all events should have timestamps, so if they don't, we'll need to file an issue/bug against the source to make sure they get put in...